decidim-pages/app/views/decidim/pages/application/show.html.erb

```diff
@@ -1,3 +1,9 @@
 <% content_for(:title, translated_attribute(@page.title)) %>
 
 <%== translated_attribute @page.body %>
+
+<%= content_for :expanded do %>
+  <% if @page.commentable? %>
```

Comment:
No: `decidim-comments` is an explicit dependency of `decidim-pages`. Instead of coupling `decidim-comments` to every possible commentable resource, I believe it's better to invert that relationship and let resources do that.